### PR TITLE
Accessibility focus-visible default halo

### PR DESCRIPTION
**Pull request by blackfalcon**
# Alaska Airlines Pull Request

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

**Resolves:** #182 

## Summary:

_Please summarize the scope of the changes you have submitted, what the intent of the work is and anything that describes the before/after state of the project._

This PR adds a new focus-visible feature to WCSS. At the same time,
the legacy focus-visible solution has been deprecated.

The use of the Sass variable to change output was not well adopted.

In order NOT to force this update into all UI scenarios as there may be
local code to address this in some cases, this feature has been added as
an OPT-IN feature.

See inline documentation for info related to update and install
instructions.

Changes to be committed:
modified:   README.md
modified:   src/_core.scss
new file:   src/_focus-visible.scss
new file:   src/bundled/essentials+fv.scss
modified:   src/elementDemoStyles/elementDemoStyles.scss

See demo: https://auro-a11y-tab-fix.surge.sh/

Screen Shot 2024-02-28 at 6 38 17 PM

## Type of change:

Please delete options that are not relevant.

- [x] New capability
- [ ] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [ ] Other (please elaborate)

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

**Comment by leeejune** [comment]
@blackfalcon Would this work?
```
border: solid 1px var(--ds-color-border-active-default);
border-radius: var(--ds-border-radius);
```

**Comment by blackfalcon** [comment]
@leeejune 

I am recommending that we DO NOT use border-radius. In many cases this halo tightly surrounds some content and with rounded corners, it is too close to the content. 

Screen Shot 2024-02-08 at 4 34 21 PM
Screen Shot 2024-02-08 at 4 38 26 PM

**Comment by leeejune** [comment]
> @leeejune
> 
> I am recommending that we DO NOT use border-radius. In many cases this halo tightly surrounds some content and with rounded corners, it is too close to the content.

@blackfalcon Sure, then the blue border should work.

**Review by leeejune — APPROVED** [review]

File: tests/styleTest.scss [hunk 1]
```diff
@@ -4,6 +4,18 @@
...
+/* --------end core import ------------- */
+
+/* --------start focus visible import ------------- */
+@import './../src/focus-visible';
+/* --------end focus visible import ------------- */
+
+/* --------start essentials import ------------- */
+@import './../src/essentials';
+/* --------end essentials import ------------- */
+
```
Comment (jason-capsule42) [inline tests/styleTest.scss:18]: Why not just import the new essentials+fv file here? It's what the updated README instructs users to do so it seems relevant for us to do the same in the testing.

Comment (blackfalcon) [reply tests/styleTest.scss:18]: The essentials+fv.scss file is part of the bundled output. This file is a test of the individual components being brought in to evaluate CSS output. 

I mean, we _could_ put essentials+fv.scss in there, but like I said, the point of this file is to see how the individual parts render into the final CSS. 

I assume you are referring to [this line](https://github.com/AlaskaAirlines/WebCoreStyleSheets/pull/183/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R71). Gawd... that's actually all wrong. I am going to have another commit that fixes that.

File: README.md [hunk 2]
```diff
@@ -104,5 +104,3 @@ The `watch` command will run a Sass linter, process a test file from Sass to CSS
 If additional selectors or scenarios are needed, please update the `./scripts/testBuild.scss` file to test your code.
 
 Running the `serve` command will open the Sassdoc view. Please review all changes as Sassdoc produces all documentation.
-
-WCSS fully supports idiomatic CSS, be sure to review the [auro docs site](https://auro.alaskaair.com/webcorestylesheets/idiomatic-css) for more information.
```
Comment (jason-capsule42) [inline README.md:102]: Does that mean this needs to be removed?

https://auro.alaskaair.com/css/idiomatic-css

**Review by jason-capsule42 — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (blackfalcon) [reply README.md:102]: Moved the conversation about the removal of this content. A larger discussion that has little impact on this PR. We can easily put this back if the resolution calls fro that action.

**Pull request merged**